awx/ui_next/src/screens/Template/WorkflowJobTemplateAdd/WorkflowJobTemplateAdd.test.jsx

```diff
@@ -63,16 +70,49 @@ describe('<WorkflowJobTemplateAdd/>', () => {
 
   test('calls workflowJobTemplatesAPI with correct information on submit', async () => {
     await act(async () => {
-      await wrapper.find('WorkflowJobTemplateForm').invoke('handleSubmit')({
-        name: 'Alex',
-        labels: [{ name: 'Foo', id: 1 }, { name: 'bar', id: 2 }],
-        organizationId: 1,
+      wrapper.find('input#wfjt-name').simulate('change', {
+        target: { value: 'Alex', name: 'name' },
       });
+
+      wrapper
+        .find('LabelSelect')
+        .find('SelectToggle')
+        .simulate('click');
+    });
+
+    wrapper.update();
+
+    act(() => {
+      wrapper
+        .find('SelectOption')
+        .find('button')
+        .at(2)
```

Comment:
Is there a way to make this select more definitive than "at(2)", maybe using button[aria-label="Foo"]?